code/modules/reagents/chemistry/reagents/impure_reagents/impure_medicine_reagents.dm

```diff
@@ -0,0 +1,412 @@
+			owner.adjustOxyLoss(-0.5)
+
+////// C2 medications
+//// Helbital
+
+//Inverse:
+/datum/reagent/inverse/helgrasp
+	name = "Helgrasp"
+	description = "This rare and forbidden concoction is thought to bring you closer to the grasp of the Norse goddess Hel."
+	metabolization_rate = 1*REM //This is fast
+	tox_damage = 0.25
+	ph = 14
+	//Compensates for delta_time lag by spawning multiple hands at the end
+	var/lag_compensate = 0
+
+//Warns you about the impenting hands
+/datum/reagent/inverse/helgrasp/on_mob_add(mob/living/L, amount)
+	to_chat(L, "<span class='hierophant'>You hear laughter as malevolent hands apparate before you, eager to drag you down to hell...! Look out!</span>")
+	playsound(L.loc, 'sound/chemistry/ahaha.ogg', 80, TRUE, -1) //Very obvious tell so people can be ready
+	. = ..()
+
+//Sends hands after you for your hubris
+/datum/reagent/inverse/helgrasp/on_mob_life(mob/living/carbon/owner, delta_time, times_fired)
+	spawn_hands(owner)
+	lag_compensate += 1 - max(delta_time, 1)
+	. = ..()
+
+/datum/reagent/inverse/helgrasp/proc/spawn_hands(mob/living/carbon/owner)
+	//Adapted from the end of the curse - but lasts a short time
+	var/grab_dir = turn(owner.dir, pick(-90, 90, 180, 180)) //grab them from a random direction other than the one faced, favoring grabbing from behind
+	var/turf/spawn_turf = get_ranged_target_turf(owner, grab_dir, 8)//Larger range so you have more time to dodge
+	if(!spawn_turf)
+		return
+	new/obj/effect/temp_visual/dir_setting/curse/grasp_portal(spawn_turf, owner.dir)
+	playsound(spawn_turf, 'sound/effects/curse2.ogg', 80, TRUE, -1)
+	var/obj/projectile/curse_hand/hel/hand = new (spawn_turf)
+	hand.preparePixelProjectile(owner, spawn_turf)
+	hand.fire()
+
+/datum/reagent/inverse/helgrasp/on_mob_delete(mob/living/owner)
+	var/hands = 0
+	while(lag_compensate > hands)
+		spawn_hands(owner)
+		hands++
+	. = ..()
+
+////libital
+
+//Impure
+
+//Simply reduces your alcohol tolerance, kinda simular to prohol
+/datum/reagent/impurity/libitoil
+	name = "Libitoil"
+	description = "Temporarilly interferes a patient's ability to process alcohol."
+	chemical_flags = REAGENT_DONOTSPLIT
+	ph = 13.5
+	liver_damage = 0.1
+	addiction_types = list(/datum/addiction/medicine = 4)
+
+/datum/reagent/impurity/libitoil/on_mob_add(mob/living/L, amount)
+	. = ..()
+	var/mob/living/carbon/carbmob = L
+	if(!carbmob)
+		return
+	var/obj/item/organ/liver/this_liver = carbmob.getorganslot(ORGAN_SLOT_LIVER)
+	this_liver.alcohol_tolerance *= 2
+
+/datum/reagent/impurity/libitoil/on_mob_delete(mob/living/L)
+	. = ..()
+	var/mob/living/carbon/carbmob = L
+	if(!carbmob)
+		return
+	var/obj/item/organ/liver/this_liver = carbmob.getorganslot(ORGAN_SLOT_LIVER)
+	this_liver.alcohol_tolerance /= 2
+
+
+////probital
+
+/datum/reagent/impurity/probital_failed//Basically crashed out failed metafactor
+	name = "Mitogen Metabolic Inhibition Factor"
+	description = "This enzyme catalyzes crashes the conversion of nutricious food into healing peptides."
```

Comment:
"catalyzes crashes"

Reply:
is there a reason why we're not just using the actual mitogen factor chem here?

Reply:
This is the fail state - so it creates peptides that cause damage instead of healing it when exposed to nutriment. The mitogen factor is the inverse of probital (between 35% - 50% will give mitogen factor)